app/views/docs/authentication-magic.phtml

```diff
@@ -0,0 +1,196 @@
+<p>
+   Magic URL is a password-less way to authenticate users. 
+   When a user logs in, they will receive an email with a "magic" link that contains a secret used to log in the user.
+   The user can simply click the link to be logged in.
+</p>
+
+<h2><a href="#init" id="init">Send Email</a></h2>
+
+<p>
+    Initialize the log in process with the <a href="/docs/client/account#accountCreateMagicURLSession">Create Magic URL Session</a> route.
+    If the email has never been used, a <b>new user ID is generated</b>, then the user will receive an email. 
```

Comment:
> new user ID is generated
Buuut only if they pass `unique()`. If they provide custom ID, user with that ID will try to be created (unless its already used by other user)

Reply:
Done